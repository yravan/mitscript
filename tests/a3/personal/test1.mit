a = "res: " + 5;
print(a);

a = 5 + " is the res";
print(a);

a = "res is: " + true;
print(a);

a = false + " is the res";
print(a);

a = "a" + "b" + "c" + "d";
print(a);

a = "res: " + None;
print(a);

a = None + " is the res";
print(a);

f = fun() {};
a = "fun is: " + f;
print(a);

a = f + " is fun";
print(a);

a = "Builtin check " + print;
print(a);

r = {
  a: 5;
  c: true;
  b: f;
};
a = "Record res is: " + r;
print(a);

a = r  + " is the other way";
print(a);
