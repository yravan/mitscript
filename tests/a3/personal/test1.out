res: 5
5 is the res
res is: true
false is the res
abcd
res: None
None is the res
fun is: FUNCTION
FUNCTION is fun
Builtin check FUNCTION
Record res is: {a:5 b:FUNCTION c:true }
{a:5 b:FUNCTION c:true } is the other way
